golang/config.yml

```diff
@@ -1,19 +1,22 @@
 test_config: &test_config
   cmd:
     - aws --version
-    - gitleaks --version
+    - gitleaks version
     - go version
     - golangci-lint --version
     - migrate -version
     - mockgen -version
     - modd --version
     - rsync --version
     - swagger version
-    - testfixtures --version
+    # testfixtures arm64 binary does not exist
+    #- testfixtures --version
```

Comment:
Remove it from the build script, and specify that we've removed it in the changelog

Reply:
Up :)